### ISSUE TITLE
Mapcode recognized as phone number

### ISSUE THREAD

**Issue by thany** [mapsme/omim#11342]
Consider this text in a bookmark description:
```
Mapcode: 592 319 072*14
```
The app will underline the _592 319 072_ part. This is useless, because it's not a phone number, and I verified: tapping it makes my phone try to call that number.

If you don't know, Denso Mapcode is a proprietary "simplified" mapping scheme for Japanese car natsavs.

I would prefer if the app didn't auto-underline phone numbers at all in the description box. Maybe you can add a setting for it.

**Comment by burivuh** [comment]
Hi!
OS version and a phone model, please.
Thanks.

**Comment by thany** [comment]
Galaxy S9, Android 8.1

Why is this needed? Is this not handled by the app?

**Comment by burivuh** [comment]
No, we use system functions to render html content.

**Comment by thany** [comment]
But it isn't html... It's just plain text, isn't it?

**Comment by burivuh** [comment]
No, our descriptions supports HTML, so every description is treated that way.

**Comment by thany** [comment]
Okay, so if it's HTML, you shouldn't magically turn things into links. HTML can do that already.

On a sidenote: What masochist types in HTML on their phone? I would much prefer markdown, which requires a lot fewer characters from the secondary keyboard and can fully serve (what I assume are) the usecases for HTML is a description. But that's a different story.

**Comment by burivuh** [comment]
1. As I've said, we do not create links directly. So we need to investigate.
2. There's plenty of people that creates bookmarks on web and then import them into Maps.Me

**Comment by thany** [comment]
Aaaand..?

**Comment by thany** [comment]
Aaaaaaan..?

**Comment by thany** [comment]
Aaaaaaaaaaaaaaaaaaand..?

**Comment by thany** [comment]
Also I'm on a Galaxy S9 still, but upgraded to Android 10. Issue persists.
